js/interactive-examples.js

```diff
@@ -0,0 +1,61 @@
+import phpBinary from "/js/php-web.mjs";
+
+function createOutput(output) {
+  const container = document.createElement('div');
+  container.classList.add('screen', 'example-contents');
+  const title = document.createElement('p');
+  title.innerText = 'Output: ';
+  container.appendChild(title)
+  const div = document.createElement('div');
+  div.classList.add('examplescode');
+  container.appendChild(div);
+  const pre = document.createElement('pre');
+  pre.classList.add('examplescode');
+  pre.innerText = output;
+  div.appendChild(pre)
+  return container;
+}
+
+async function main() {
+  const buffer = [];
+	const {ccall} = await phpBinary({
```

Comment:
```suggestion
  const {ccall} = await phpBinary({
```

(mixed tabs and spaces)